### ISSUE TITLE
Test PSNR/SSIM question

### ISSUE THREAD

**Issue by MengXinChengXuYuan** [MTlab/cnn_image_enhance#5]
Hi thank you for your great work first!

I followed all your instructions in read me, using the DPED dataset, the iphone-cannon part (seems you are using the part only according to the dataset list provided)
I didn't change anything and the training goes well, I can get around 22 psnr on the training set. However I only get around 11, 12 maximum on the test set ...

I modified the provided run.py to fit the enhancement network, converting the input image according to the training procedure, the result for normal light condition images are ... I'd say so so, cause the test psnr I get is only 12, which a little color distortion, tends to be brown-gray.
However when the input image is in very low light condition, it become even worse, the output is entirely gray(and a little brown)

What could be wrong? The reported psnr on DPED benchmark is 21.89, how can I reproduce that result? How many iters are requested to get that result? Or can anyone else get a good psnr on the test set? I wonder if there is something wrong, cause the test psnr stay around 11.5 in very early iters.
I noticed that the provide weights for those loss proposed aren't configured exactly the same as in the paper. So I also tried to make them configured exactly the same, the result still stay the same ...

Thank you very much if you can give me any hints !

**Comment by Oktai15** [comment]
@MTlab @gamewang123 @MengXinChengXuYuan I have the same problem. Any idea what problem is?

**Comment by wxkb** [comment]
> 嗨，谢谢您的出色工作！
> 
> 我按照您的所有说明进行了阅读，使用了DPED数据集，iphone-cannon部分（似乎仅根据提供的数据集列表使用了该部分），
> 我没有做任何更改，并且训练进行得很好，我可以绕开训练集上的22 psnr。但是我在测试集上最多只能得到11、12左右...
> 
> 我修改了提供的run.py以适应增强网络，根据训练过程转换了输入图像，正常光线条件图像的结果是...这么说，导致我得到的测试psnr只有12稍有颜色失真的颜色往往是棕灰色。
> 但是，当输入图像处于非常弱的光线条件下时，甚至会变得更糟，输出图像将完全变成灰色（有点棕色）
> 
> 有什么事吗 DPED基准上报告的psnr为21.89，如何重现该结果？需要多少次迭代才能获得该结果？还是其他人可以在测试仪上获得良好的psnr？我想知道是否有问题，导致测试psnr在非常早的迭代中保持在11.5左右。
> 我注意到，建议的那些损失的权重配置与本文中的配置完全不同。所以我也尝试使它们的配置完全相同，结果仍然保持不变...
> 
> 如果可以给我任何提示，非常感谢！

**Comment by wxkb** [comment]
> 嗨，谢谢您的出色工作！
> 
> 我按照您的所有说明进行了阅读，使用了DPED数据集，iphone-cannon部分（似乎仅根据提供的数据集列表使用了该部分），
> 我没有做任何更改，并且训练进行得很好，我可以绕开训练集上的22 psnr。但是我在测试集上最多只能得到11、12左右...
> 
> 我修改了提供的run.py以适应增强网络，根据训练过程转换了输入图像，正常光线条件图像的结果是...这么说，导致我得到的测试psnr只有12稍有颜色失真的颜色往往是棕灰色。
> 但是，当输入图像处于非常弱的光线条件下时，甚至会变得更糟，输出图像将完全变成灰色（有点棕色）
> 
> 有什么事吗 DPED基准上报告的psnr为21.89，如何重现该结果？需要多少次迭代才能获得该结果？还是其他人可以在测试仪上获得良好的psnr？我想知道是否有问题，导致测试psnr在非常早的迭代中保持在11.5左右。
> 我注意到，建议的那些损失的权重配置与本文中的配置完全不同。所以我也尝试使它们的配置完全相同，结果仍然保持不变...
> 
> 如果可以给我任何提示，非常感谢！

您好，我也在复现这篇论文但碰到了一些问题，您能指导下吗

**Comment by zhangqizky** [comment]
The same problems...Do you guys have any solutions? In my case, training psnr is around 37, but test psnr is around 12....
